CHANGELOG.md

```diff
@@ -2,6 +2,48 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## 2.0.0 (2022-01-24)
+
+
+### ⚠ BREAKING CHANGES
+
+* **metrics:** plugin configuration schema overhauled, default metric name of `registry_requests`
+renamed to `registry_package_downloads`
+* **configuration:** configuration option `enabled` changed to `metricsEnabled`
+
+### Features
+
+* **metrics:** collect http request metrics and package download metrics ([05b1c96](https://github.com/xlts-dev/verdaccio-prometheus-middleware/commit/05b1c961fe8c22b36a872f9eeadbdec385ec7de0)), closes [#9](https://github.com/xlts-dev/verdaccio-prometheus-middleware/issues/9)
+* **metrics:** implement ability to collect default prometheus metrics ([49cc20b](https://github.com/xlts-dev/verdaccio-prometheus-middleware/commit/49cc20b937e0d82e2007eece60d0332eaca9c37b)), closes [#4](https://github.com/xlts-dev/verdaccio-prometheus-middleware/issues/4)
+* **metrics:** Initial implementation of package install/download metrics capturing and metrics endpoint ([11e8ac6](https://github.com/xlts-dev/verdaccio-prometheus-middleware/commit/11e8ac6fc89c44531a5753d5b672276174972524))
+
+
+### Bug Fixes
+
+* **configuration:** configuration option `enabled` changed to `metricsEnabled` ([cc725e9](https://github.com/xlts-dev/verdaccio-prometheus-middleware/commit/cc725e9abbe4c8f1d81b145131747f1abf7f51d7))
+* **metrics:** fix issue where package download counters were not fully accurate ([86546bf](https://github.com/xlts-dev/verdaccio-prometheus-middleware/commit/86546bf30eb007103bf343013e426150a73efb53)), closes [#7](https://github.com/xlts-dev/verdaccio-prometheus-middleware/issues/7)
+* **metrics:** Only collect metrics for download of tarball files. Metrics were previously collected for any GET request that looked like it was for a package.json or tarball install but this was less reliable as there were was no way to guarantee the request coming in was for an actual package.json (e.g. browser requests for favicon.ico). Also, Verdaccio would only hand off requests that generate a 401/403 to the middelware for non-tarball requests so metrics could be misleading. ([67caa0f](https://github.com/xlts-dev/verdaccio-prometheus-middleware/commit/67caa0f2e733966af9ece23ab649879378bd28e4))
+
+## 2.0.0 (2022-01-24)
```

Comment:
It looks like it duplicated the 2.0.0 release notes?

Reply:
This was from doing a `git commit --amend` and `npm run release` without reverting the file first from the previous release. Fixed.